PLAYER/SetAllRandomPedsFlee.md

```diff
@@ -8,6 +8,7 @@ ns: PLAYER
 void SET_ALL_RANDOM_PEDS_FLEE(Player player, BOOL toggle);
 ```
 
+Sets whether all random peds will run away from player if one attacks or harasses them (bool=true), or some peds can stand up for themselves (bool=false).
 
 ## Parameters
```

Comment:
Mind giving the parameters a description to wrap this up? Thanks!

Reply:
> Mind giving the parameters a description to wrap this up? Thanks!

Done!

Reply:
Great, if you can, do please take a look at @gottfriedleibniz [comment](https://github.com/citizenfx/natives/pull/779#issuecomment-1163276301), he makes a great point.

Reply:
Ok=)